src/main/java/core/basesyntax/dao/FruitTransaction.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.dao;
+
+public interface FruitTransaction {
```

Comment:
it should be just POJO, see task description. As a mock db it's enough to have Storage class with only public static Map<String, Integer>, no setters and getters, and just use it where need